swagger/public_v1/api_spec.json

```diff
@@ -1384,8 +1384,8 @@
             "style": "form",
             "explode": false,
             "required": false,
-            "example": "provider.provider_name,name",
-            "description": "Field(s) to sort the courses by."
+            "example": "name",
```

Comment:
this file is auto-generated so this need to be changed at the source spec/api/providers_spec.rb otherwise it'll get overwritten again

Reply:
nice spot